modules/n1ql/pages/n1ql-language-reference/alterindex.adoc

```diff
@@ -195,30 +240,45 @@ WITH {"action": "move", "nodes":
 ----
 ====
 
-.Removing an extra replica
+.Increasing the number of replicas
```

Comment:
There are two possibilities. 
(1) If they want, user can specify the "nodes" parameter in the "WITH" clause in ALTER INDEX statement.
(2) If they don't want, user can choose not to specify any "nodes".

Both of these are valid scenarios for the user. 

When "nodes" is specified, the new replica will be placed on the nodes from the list specified by the user.
When "nodes" is not specified, the replica can be placed on any of the available indexer nodes in the cluster.

Note: In both of these cases, no single indexer node will host 2 replicas of the same index (or same partition of the same index). Also, replicas will get distributed across the available server groups.

I suggest we should have two examples for "Increasing the number of replicas", one without "nodes" parameter and one with "nodes" parameter.

Reply:
Done